Public/Src/Cache/ContentStore/Grpc/ContentStore.proto

```diff
@@ -196,6 +198,15 @@ enum CopyCompression {
     gzip = 1;
 }
 
+// PushFile
+message PushFileRequest {

```

Comment:
Should be possible to re-use CopyFileResponse type. Just re-name to FileTransferChunk or similiar.

Reply:
In here I'm not using indexes. Besides, I think there is value in being able to modify both independently, particularly since there is virtually no cost in having these as separate response type.

---
In reply to: [343805723](https://github.com/microsoft/BuildXL/pull/1141#discussion_r343805723) [](ancestors = 343805723)